sympy/algebras/tests/test_quaternion.py

```diff
@@ -25,17 +25,12 @@ def test_quaternion_construction():
 def test_quaternion_complex_real_addition():
     a = symbols("a", complex=True)
     b = symbols("b", real=True)
-    # This symbol is not complex:
-    c = symbols("c", commutative=False)
 
     q = Quaternion(x, y, z, w)
     assert a + q == Quaternion(x + re(a), y + im(a), z, w)
     assert 1 + q == Quaternion(1 + x, y, z, w)
     assert I + q == Quaternion(x, 1 + y, z, w)
     assert b + q == Quaternion(x + b, y, z, w)
-    assert c + q == Add(c, Quaternion(x, y, z, w), evaluate=False)
```

Comment:
It is not necessary to totally remove these tests. It should be possible to write something like `raises(ValueError, lambda: c + q)` instead.

Reply:
Done. Thanks for reviewing again.